src/monitoring_frame_deserialization.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
```

Comment:
I would sort the definitions of the methods here in the same order they appear in the header.